README.md

```diff
@@ -32,35 +37,34 @@ You can run automated tests with `pytest`:
 pytest -svx tests
 ```
 
-## Development
-
-The plugin is still under development. If you would like to contribute, install the package in editable mode with the development dependencies:
-
-```sh
-pip install -e .[dev] --index-url https://gitlab.mpcdf.mpg.de/api/v4/projects/2187/packages/pypi/simple
-```
 
```

Comment:
I think ideally we should have some instructions pointing to using the package in `editable` mode when developing. It will be important when having new developers onboard.

Reply:
What do you mean? I thought these lines were pretty much repetition of the "Install the dependencies" part.

Reply:
There is a flag `-e` that differs, which installs the package in editable mode. So that when you make some changes to the code in the package, it reflects in the installed package as well.
https://setuptools.pypa.io/en/latest/userguide/development_mode.html

Reply:
Ah true, you are right. My bad, I will restore the Development section.